tracer-hystrix/src/test/java/org/zalando/tracer/hystrix/TracerConcurrencyStrategyTest.java

```diff
@@ -64,4 +84,18 @@ protected String run() {
 
     }
 
+    private final class TimesOut extends HystrixCommand<String> {
+
+        TimesOut() {
+            super(HystrixCommandGroupKey.Factory.asKey("ExampleGroup"));
```

Comment:
Timeouts are never configured anywhere, but I guess based on https://github.com/Netflix/Hystrix/wiki/Configuration#executionisolationthreadtimeoutinmilliseconds, it's enabled by default to 1000ms. Correct?